get_validator.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+
+wget https://github.com/hapifhir/org.hl7.fhir.core/releases/download/5.6.54/validator_cli.jar --no-check-certificate
```

Comment:
I'd rather not default to turning of certificate checking. Could we have this version in here commented out so that the people who need it can use it but check certificates by default?

Reply:
Yes, fixed the script as requested.